### DISCUSSION TITLE
unable to navigate to Relative URLs and static files

### DISCUSSION THREAD

**Discussion by thecodinghabits** [codesandbox/sandpack#531] [Q&A]
while using sandpack, I stumbled upon an issue where I was not able to navigate to other static html files using hyper links. A similar problem has been raised by Gitlabs. Is there any way to fix this.

https://gitlab.com/gitlab-org/gitlab/-/issues/197206

**Comment by danilowoz** [comment]
Hey @thecodinghabits, thanks for reporting this issue. I'm not I understood the problem correctly:
- Aren't you able to load static files?
- Or aren't you able to navigate to them? 

It's weird because I don't think Sandpack supports static files like images, for example. So, could you please provide a reproducible example so I can debug it? 

Thanks again
